backpack/core/derivatives/basederivatives.py

```diff
@@ -486,6 +486,7 @@ def sqrt_hessian_sampled(
         g_out: Tuple[Tensor],
         mc_samples: int = 1,
         subsampling: List[int] = None,
+        use_autograd: bool = False,
```

Comment:
`use_autograd` should move to the `NLLBaseDerivatives` class and not show up in this interface, because it is only a relevant concept for `NLL` losses.